### PR TITLE
update for new zend\hydrator component

### PR DESCRIPTION
**Pull request by prolic**

### CODE REVIEW

**Comment by prolic** [comment]
Question: Do you also want to rename DoctrineModule\Stdlib\Hydrator\DoctrineObject => DoctrineModule\Hydrator\DoctrineObject ?

**Comment by prolic** [comment]
Needed to drop 5.4 support to get the tests running again. PHP 5.4 is also not support by PHP-Internals anymore.

**Comment by Ocramius** [comment]
> Question: Do you also want to rename DoctrineModule\Stdlib\Hydrator\DoctrineObject => DoctrineModule\Hydrator\DoctrineObject ?

Not a good idea

**Comment by prolic** [comment]
weird that the build succeeded. zend-mvc 2.6 required zend-hydrator 1.0, should be incompatible with 2.0.

**Comment by gianarb** [comment]
@Ocramius it is a BC Break? :) I can merge it during 1.0 release

File: tests/Bootstrap.php [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?php
 chdir(dirname(__DIR__));
 
-$loader = require_once('vendor/autoload.php');
+$loader = require ('vendor/autoload.php');
```
Comment (Bilge) [inline tests/Bootstrap.php]: Why?

**Comment by alcaeus** [comment]
Closing since the pull request is rather old and the patch doesn't apply cleanly. I've checked whether the class in question is still used, which it isn't, so I also assume that this was cleaned up in the meantime. If not, please let me know.

**Pull request closed without merging**